herbert_core/utilities/general/is_def.m

```diff
@@ -0,0 +1,7 @@
+function ok = is_def(name)
```

Comment:
I would find `is_defined` a clearer function name.

Reply:
Yes, I originally had `is_defined`, I found it a little verbose in the way `is_defined` was sometimes used. Easy to change though.